src/mapsector.h

```diff
@@ -50,7 +50,7 @@ class MapSector
 		return m_pos;
 	}
 
-	MapBlock * getBlockNoCreateNoEx(s16 y);
+	MapBlock * getBlockNoCreate(s16 y);
```

Comment:
```suggestion
	MapBlock *getBlockNoCreate(s16 y);
```

Please also fix this in the following lines, if possible.